client/resource_group/controller/controller.go

```diff
@@ -465,7 +465,9 @@ type groupCostController struct {
 
 	mu struct {
 		sync.Mutex
-		consumption *rmpb.Consumption
+		consumption   *rmpb.Consumption
+		storeCounter  map[uint64]*rmpb.Consumption
```

Comment:
How add adding a lock? The logic for `storeCounter` and `globalCounter ` is independent of `consumption`

Reply:
but they almost use the lock at the same time

Reply:
This a hot path. I guess two locks will help get better performance. But maybe  it's not the bottleneck currently